http/config.go

```diff
@@ -81,14 +81,18 @@ const (
 type AuthType string
 
 const (
-	// BearerTokenAuth specifies that a bearer token is required.
+	// BearerTokenAuth specifies that a legacy bearer token (v1) authentication is in use
 	BearerTokenAuth AuthType = "token"
+
+	// BearerTokenAuthV2 specifies the latest version of bearer token authention
+	BearerTokenAuthV2 = "token_v2"
 )
 
 // AuthConfig contains the configuration for authentication for an HTTP interface.
 type AuthConfig struct {
 	// Type specifies the type of authentication required for the interface.
 	Type AuthType `koanf:"type"`
+	AuthorizedKeysPath string `koanf:"authorizedkeyspath"`
```

Comment:
or could be;

`authorizedkeys.path` and `authorizedkeys.text` (although `text` is ugly and for future debate).

Reply:
Also; add it to  `http/cmd/cmd.go:FlagSet()` so it gets documented in the CLI documentation (generate docs, `make gen-docs`). Although it will be just for the `default` HTTP interface, at least it's documented that way.

And documentation in general (extend `docs/pages/integrating/api-authentication.rst`).

Reply:
[discussed in person]

We want to make it very clear whether this is meant to be the contents of an authorized_keys file or the path to one. Not simply to be pedantic today, but to allow for setting the *contents* of an authorized_keys file in a separate config entry at a later point without running into the problem of creating severe confusion in our config files.

When in doubt...spell it out =).